lib/ar2dto/has_dto.rb

```diff
@@ -24,6 +24,11 @@ def has_dto
 
         model.include Model::InstanceMethods
       end
+
+      # @api public
+      def to_dto
```

Comment:
Should we add any text in the README related to this usage?

Reply:
there is already something in the README 😄  even before it existed 🤣